requirements.txt

```diff
@@ -1,26 +1,26 @@
 # MISC
-PyYAML==5.4
+PyYAML==6.0
 
 # ASYNC
 uvloop==0.17.0
 
 # HTTP
-requests==2.22.0
+requests==2.28.0
 aiohttp>=3.7.4
 
 # SEARCH
-fuzzywuzzy[speedup]==0.17.0
+fuzzywuzzy[speedup]==0.18.0
 
 # API's
-discord.py[voice]==2.0.1
+discord.py[voice]==2.1.1
 
 # CHARTS
 matplotlib>=3.1.1
 pandas>=1.0.0
-emoji==0.5.4
+emoji==2.2.0
 
 # TEXT FORMATTING
-titlecase==0.12.0
+titlecase==2.4.0
```

Comment:
Same here

Reply:
Tested, works fine.

For ref, `titlecase` is used in Autorole for Other Games roles display.